test/Microsoft.AspNetCore.SignalR.Tests/EchoEndPoint.cs

```diff
@@ -11,7 +11,7 @@ namespace Microsoft.AspNetCore.SignalR.Tests
 {
     public class EchoEndPoint : EndPoint
     {
-        public async override Task OnConnectedAsync(ConnectionContext connection)
+        public override async Task OnConnectedAsync(ConnectionContext connection)
```

Comment:
Resharper or something? It's fine (better, even), just curious :)

Reply:
Yes. Do most MS devs not use it? I see a lot of RS suggestions throughout signalr code.

Reply:
Plenty of folks at MS use it (EF team uses it quite a bit). No one on the SignalR team uses it that I know of, but there's no problem with it. We can talk about trying to resolve some of the suggestions. It's certainly OK to throw these small fixes in as you're modifying a file.